ContentPatcher/docs/i18n/zh/author-guide/tokens.md

```diff
@@ -0,0 +1,2011 @@
+}
+
+// 玩家既无铁匠也无宝石学家职业
+"When": {
+   "HasProfession |contains=Blacksmith, Gemologist": "false"
+}
+```
+
+#### `valueAt`
+`valueAt`参数获取令牌中指定位置的值（首项为0）。若索引超出范围，返回空列表。
+
+顺序取决于令牌，可通过[`patch summary unsorted`命令](troubleshooting.md#summary)查看。如`ChildNames`有固定顺序，大多数如`HasFlag`按游戏数据顺序排列，可能随存档变化。
+
+例如
+
+<table>
+  <tr>
+    <th>令牌</th>
+    <th>值</th>
+  </tr>
+  <tr>
+    <td><code>{{ChildNames}}</code></td>
+    <td><code>Angus, Bob, Carrie</code></td>
+  </tr>
+  <tr>
+    <td><code>{{ChildNames |valueAt=0}}</code></td>
+    <td><code>Angus</code></td>
+  </tr>
+  <tr>
+    <td><code>{{ChildNames |valueAt=1}}</code></td>
+    <td><code>Bob</code></td>
+  </tr>
+  <tr>
+    <td><code>{{ChildNames |valueAt=2}}</code></td>
+    <td><code>Carrie</code></td>
+  </tr>
+  <tr>
+    <td><code>{{ChildNames |valueAt=3}}</code></td>
+    <td><em>空列表</em></td>
+  </tr>
+</table>
+
+负索引从列表末尾开始，-1为最后一项：
+
+<table>
+  <tr>
+    <th>令牌</th>
+    <th>值</th>
+  </tr>
+  <tr>
+    <td><code>{{ChildNames}}</code></td>
+    <td><code>Angus, Bob, Carrie</code></td>
+  </tr>
+    <td><code>{{ChildNames |valueAt=-1}}</code></td>
+    <td><code>Carrie</code></td>
+  </tr>
+    <td><code>{{ChildNames |valueAt=-2}}</code></td>
+    <td><code>Bob</code></td>
+  </tr>
+    <td><code>{{ChildNames |valueAt=-3}}</code></td>
+    <td><code>Angus</code></td>
+  </tr>
+    <td><code>{{ChildNames |valueAt=-4}}</code></td>
+    <td><em>空列表</em></td>
+  </tr>
+</table>
+
+<a name="custom-input-value-separator"></a>
+
+### 自定义参数分割符号 
+默认输入参数以逗号分隔，但有时需允许逗号出现在值中。可使用`inputSeparator`参数指定不同分隔符（可为一个或多个字符）。
+
+例如，允许随机对话中的分隔符不为逗号：
+
+```json
+"Entries": {
+   "fri": "{{Random: Hey, how are you? @@ Hey, what's up? |inputSeparator=@@}}"
+}
+```
+
+**注意：** 应避免在分隔符中使用`{}|=:`，即使理论上技术有效。分隔符可能与令牌语法冲突，可能随Content Patcher版本改进。
+<a name="randomization"></a>
+
+## 随机 
+### 概述
+可使用`Random`令牌来随机化：
+```js
+{
+   "Action": "Load",
+   "Target": "Characters/Abigail",
+   "FromFile": "assets/abigail-{{Random:hood, jacket, raincoat}}.png"
+}
+```
+
+可使用固定键保持多个`Random`令牌同步（详见下文）：
+```js
+{
+   "Action": "Load",
+   "Target": "Characters/Abigail",
+   "FromFile": "assets/abigail-{{Random:hood, jacket, raincoat |key=outfit}}.png",
+   "When": {
+      "HasFile:assets/abigail-{{Random:hood, jacket, raincoat |key=outfit}}.png": true
+   }
+}
+```
+
+此令牌是动态的，可能有意外冲突，详见下文。
+
+### 独特属性
+`Random`令牌是……
+
+<ol>
+<li>
+
+**动态的。** Random令牌在每次加载补丁时重新选择，通常在新一天开始时。随机更新速率基于游戏+游戏内日期+输入字符串，因此重载存档不会改变已选随机值。
+
+</li>
+<li>
+
+**独立的。** 每个`Random`变化是独立的。例如：
+
+* 若补丁有多个`Target`，每个目标可能得到不同的`Random`值。
+* 若`FromFile`字段的`Random`令牌在`HasFile`字段中使用，可能因不同选择导致文件加载失败。
+
+需同步多个`Random`时，使用 _固定键_ 。
+</li>
+<li>
+
+**公平的。** 每个选项概率相等。可通过重复值调整概率。例如，'red'出现概率是'blue'的两倍：
+```js
+{
+   "Action": "Load",
+   "Target": "Characters/Abigail",
+   "FromFile": "assets/abigail-{{Random:red, red, blue}}.png"
+}
+```
+
+</li>
+<li>
+
+**有界的。** 选项不含令牌。例如，若所有选项为'true'或'false'，可在布尔类型的条目中使用；若为数字，可在数值类型的条目中使用。
```

Comment:
> **有界的。** 选项不含令牌。

I think this might change the meaning a bit? The original text says that a `Random` token is bounded if the arguments contain no tokens (and unbounded if they do); the translation seems to say it's always bounded and cannot contain tokens?